fastapi/models/fastapi_endpoint.py

```diff
@@ -19,6 +21,26 @@
 _logger = logging.getLogger(__name__)
 
 
+# Thread-local storage for event loops
+# Using a thread-local storage allows to have a dedicated event loop per thread
+# and avoid the need to create a new event loop for each request. It's also
+# compatible with the multi-worker mode of Odoo.
+_event_loop_storage = threading.local()
+
+
+def get_or_create_event_loop() -> asyncio.AbstractEventLoop:
+    """
+    Get or create a reusable event loop for the current thread.
+    """
+    if not hasattr(_event_loop_storage, "loop"):
+        loop = asyncio.new_event_loop()
+        loop_thread = threading.Thread(target=loop.run_forever, daemon=True)
+        loop_thread.start()
+        _event_loop_storage.loop = loop
+        _event_loop_storage.thread = loop_thread
```

Comment:
Is it necessary to store the loop thread here?

Reply:
IMO no, we don't need it